ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly-x86.yaml

```diff
@@ -21,15 +21,19 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: perfscale-rosa-network-perf-periodic
-  cron: 0 8 * * 5
+- as: perfscale-rosa-multiaz-cluster-density-v2-24nodes-periodic
+  cron: 0 8 * * 4
```

Comment:
all other tests in this file you are setting to not run, do you want to set this one to not run as well?

Reply:
we want to run cluster-density-v2 test on a cron basis and not run others on a cron basis

Reply:
should we run all of the control plane tests not just cluster-density?